CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.15.0 (2020-02-12)
+
```

Comment:
Added support for `skyux build-public-library --fullTemplateTypeChecking` which checks and validates TypeScript types used within component HTML templates.

Reply:
Done!